Automattic/vip-go-mu-plugins
# VIP Support: Remove vip_support from editable_roles for users without the vip_support role

## Description
Fixes https://github.com/Automattic/vip-go-mu-plugins/issues/1288.

## Changelog Description

### Plugin Updated: VIP Support

Remove vip_support role from editable_roles for non-vip_support users.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Create admin user and login with that user
2) Go to Users > Add New and under the "Roles" drop-down confirm that "VIP Support" is no longer listed
3) Go to Users > All Users and under the drop down of "Change role to", confirm that "VIP Support" is no longer listed
